util/cloud/gcp/gcs.cc

```diff
@@ -144,9 +144,11 @@ io::Result<TokenTtl> ParseTokenResponse(std::string&& response) {
   return result;
 }
 
+static const char kMetaDataHost[] = "metadata.google.internal";
```

Comment:
```suggestion
static constexpr string_view kMetaDataHost = "metadata.google.internal"sv;
```
Just a nit, doesn't really matter